docs/docsite/rst/userguide/teams.rst

```diff
@@ -78,10 +83,12 @@ In order to add a user to a team, the user must already be created. Refer to :re
 To remove roles for a particular user, click the disassociate (x) button next to its resource.
 
 .. image:: ../common/images/permissions-disassociate.png
+   :alt: Disassociate Button
```

Comment:
I suggest something more descriptive along these lines:
"Access tab with list of users and an arrow pointing to the disassociate button next to a user's role."